EXOSIMS/Prototypes/PlanetPhysicalModel.py

```diff
@@ -24,7 +24,7 @@ class PlanetPhysicalModel(object):
 
     _modtype = 'PlanetPhysicalModel'
 
-    def __init__(self, cachedir=None, **specs):
+    def __init__(self, cachedir=None, whichPlanetPhaseFunction='lambert', **specs):

```

Comment:
this is an ICD-level change.  Please update the ICD accordingly.  The docstring needs to also be updated to specify this input. Does it have to be a string?  Can we make it an object input such that I can pass an arbitrary method if I wanted to?

Less importantly, why 'whichPlanetPhaseFunction' instead of just 'PlanetPhaseFunction'?